recipe/build-libadios.sh

```diff
@@ -66,37 +47,38 @@ if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
     export CMAKE_ARGS="${CMAKE_ARGS} -DADIOS2_HAVE_ZFP_CUDA_EXITCODE=0"
 fi
 
-
-cmake              \
-    -S ${SRC_DIR}  \
-    -B build       \
-    ${CMAKE_ARGS}  \
+# shellcheck disable=SC2086
+cmake               \
+    $CMAKE_ARGS     \
+    -S "$SRC_DIR"   \
+    -B build        \
+    -GNinja         \
     -DCMAKE_BUILD_TYPE=Release                \
-    -DCMAKE_VERBOSE_MAKEFILE=ON               \
     -DBUILD_SHARED_LIBS=ON                    \
-    -DBUILD_TESTING=${RUN_TESTS}              \
-    -DCMAKE_CXX_STANDARD=${CXX_STANDARD}      \
-    -DCMAKE_CXX_STANDARD_REQUIRED=ON          \
-    -DCMAKE_CXX_EXTENSIONS=${CXX_EXTENSIONS}  \
-    -DADIOS2_USE_Blosc=ON                     \
+    -DBUILD_TESTING="$RUN_TESTS"              \
     -DADIOS2_USE_BZip2=ON                     \
     -DADIOS2_USE_HDF5=ON                      \
-    -DADIOS2_USE_MPI=${USE_MPI}               \
+    -DADIOS2_USE_MPI="$USE_MPI"               \
```

Comment:
Actually that is clearer & safer in bash, imho:
```suggestion
    -DADIOS2_USE_MPI=${USE_MPI}               \
```

https://github.com/azet/community_bash_style_guide

Since the argument is also quite trivial, additional quotes are ok but not really needed.

Reply:
How is using {} safer?, I can see the quotes needs to avoid expanding something with spaces.

Reply:
It's safer / clearer if someone wants to do things like `"${var_name}_static_foo"` down the road.